test/board_test.rb

```diff
@@ -1,9 +1,68 @@
 require "test_helper"
 
 class BoardTest < Minitest::Test
```

Comment:
These tests are a great example of testing expected values instead of expected behavior! Look at how much shorter and easier to read these tests are! Could you design all your objects to work this way?